servers/rendering/renderer_rd/storage_rd/material_storage.cpp

```diff
@@ -480,13 +480,20 @@ _FORCE_INLINE_ static void _fill_std140_variant_ubo_value(ShaderLanguage::DataTy
 					gui[1] = v.y;
 					gui[2] = v.z;
 					gui[3] = v.w;
-				} else {
+				} else if (value.get_type() == Variant::PLANE) {
```

Comment:
On the topic of misc fixes, I think it's likely better that this check this should be here anyway, and the final `else` would be an error.